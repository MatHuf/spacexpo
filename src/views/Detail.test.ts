import React from "react";
import renderer from "react-test-renderer";

import Details from "./Detail";

describe("<Details />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(React.createElement(Details)).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
