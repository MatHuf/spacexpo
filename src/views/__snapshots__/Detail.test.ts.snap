// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Details /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 32,
      }
    }
  >
    Details here...
  </Text>
</View>
`;
